modules/claims_api/app/swagger/claims_api/v0/form_2122_controller_swagger.rb

```diff
@@ -367,6 +369,7 @@ class Form2122ControllerSwagger
             key :name, 'X-VA-Last-Name'
             key :in, :header
             key :description, 'Last Name of Veteran being represented'
+            key :example, 'Doe'
```

Comment:
These example values are incorrect I believe. These values should be of a Veteran being represented. John Doe is not a Veteran but is representing several Veterans. This could be any of the following instead: 001, 002, 003, 004, 005, and 006 (Tamara Ellis, Janet Moore, Ralph Lee, Jess Gray, Pauline Foster, and Russell Freeman)

source for above example values: https://github.com/department-of-veterans-affairs/vets-api-clients/blob/master/test_accounts.md#representative

Reply:
Updated [here](https://github.com/department-of-veterans-affairs/vets-api/pull/6005/commits/b070d39b2ab82cbb2ed5c2d39f9c9c63704d6b31).